internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/event.go

```diff
@@ -16,14 +16,15 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
-// Schema: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-buffer.json
-const componentID = keboola.ComponentID("keboola.keboola-buffer")
+// Schema: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-stream..json
```

Comment:
```suggestion
// Schema: https://github.com/keboola/event-schema/blob/main/schema/ext.keboola.keboola-stream.json
```

Reply:
No, the two dots are intentional.

Reply:
True, I though at first it is mistake, but it is correct. Here is readme https://github.com/keboola/event-schema/blob/main/README.md

Reply:
Thank you 👍